_includes/code/components/search-result-code.html

```diff
@@ -1,11 +1,12 @@
 {% capture code_preview %}
-<h6>Without metadata</h6>
+<h6>Without Metadata</h6>
```

Comment:
@anuragmundada-reisys - Getting there!

We can delete these first two examples now that we have the specific ones for the FAL and Opportunities.